libraries/NonFungibleToken/contracts/math.aes

```diff
@@ -0,0 +1,32 @@
+namespace SafeMath =
```

Comment:
Most of this is built into the VM, not sure what is the point of this library?

Reply:
~~Most~~ All(?)

Reply:
Agree with @hanssv